src/index.ts

```diff
@@ -32,18 +36,26 @@ export default async function commitlint(
   const config = { ...defaultConfig, ...userConfig };
 
   for (const commit of danger.git.commits) {
-    await lintCommitMessage(commit.message, rules, config.severity);
+    await lintCommitMessage(commit.message, rules, config);
   }
 }
 
-async function lintCommitMessage(commitMessage, rules, severity) {
+async function lintCommitMessage(commitMessage, rules, config) {
   return lint(commitMessage, rules).then((report) => {
     if (!report.valid) {
-      let failureMessage = `There is a problem with the commit message\n> ${commitMessage}`;
+      const messagePrefix =
+        config.customMessage?.prefix ||
```

Comment:
The description says "add to the outputted message" but this _replaces_ the default.

Reply:
Yes you are right, I was misleading in the description  - in the first attempt, I actually intended for the prefix to overwrite the default one and for suffix to be added in the end, but I didn't communicate it properly.